notebook/agentchat_agentops.ipynb

```diff
@@ -2,117 +2,142 @@
-    "Using [AgentOps](https://agentops.ai/) allows you to quickly and easily add agent observability and monitoring to your Autogen services."
-   ],
-   "metadata": {
-    "collapsed": false
-   },
-   "id": "5a3b20ae8f10e4e9"
+    "<img src=\"https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/banner.png\" alt=\"Description of image\" width=\"60%\"/>\n",
+    "\n",
+    "[AgentOps](https://agentops.ai/?=autogen) provides session replays, metrics, and monitoring for AI agents.\n",
+    "\n",
+    "At a high level, AgentOps gives you the ability to monitor LLM calls, costs, latency, agent failures, multi-agent interactions, tool usage, session-wide statistics, and more. For more info, check out the [AgentOps Repo](https://github.com/AgentOps-AI/agentops).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b354c068",
+   "metadata": {},
+   "source": [
+    "### Screenshots\n",
+    "<details open>\n",
+    "  <summary>Agent Dashboard</summary>\n",
+    "   <img src=\"https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/overview.png\" style=\"width: 90%;\" alt=\"Agent Dashboard\"/>\n",
+    "</details>\n",
+    "\n",
+    "<details>\n",
+    "  <summary>Session Statistics</summary>\n",
+    "   <img src=\"https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/session.png\" style=\"width: 90%;\" alt=\"Agent Dashboard\"/>\n",
+    "</details>\n",
+    "\n",
+    "<details>\n",
+    "  <summary>Session Replay</summary>\n",
+    "   <img src=\"https://raw.githubusercontent.com/AgentOps-AI/agentops/main/docs/img/replay.png\" style=\"width: 90%;\" alt=\"Agent Dashboard\"/>\n",
+    "</details>\n"
+   ]
   },
   {
    "cell_type": "markdown",
+   "id": "1731b1788d44533e",
+   "metadata": {},
    "source": [
-    "````{=mdx}\n",
-    ":::info Requirements\n",
+    "## Requirements\n",
     "Some extra dependencies are needed for this notebook, which can be installed via pip:\n",
     "\n",
     "```bash\n",
     "pip install pyautogen[agentops]\n",
-    "```\n",
-    "\n",
-    "For more information, please refer to the [installation guide](/docs/installation/).\n",
-    ":::\n",
-    "````"
-   ],
-   "metadata": {
-    "collapsed": false
-   },
-   "id": "4f9b51b9ec511742"
+    "```"
+   ]
   },
   {
    "cell_type": "markdown",
+   "id": "38182a5296dceb34",
+   "metadata": {},
    "source": [
-    "### Adding AgentOps to an existing Autogen service.\n",
+    "## Adding AgentOps to an existing Autogen service.\n",
     "It's as simple as adding two lines of code!"
-   ],
-   "metadata": {
-    "collapsed": false
-   },
-   "id": "146e926be404dcd2"
+   ]
   },
   {
    "cell_type": "code",
+   "execution_count": null,
+   "id": "b6dc3b19889ad683",
+   "metadata": {},
    "outputs": [],
    "source": [
-    "%pip install agentops"
-   ],
-   "metadata": {
-    "collapsed": false
-   },
-   "id": "50da532c9de725f1",
-   "execution_count": null
+    "%pip install pyautogen[agentops]"
```

Comment:
they want us to remove our optional dependency from autogen

Reply:
Yeah this ^ @areibman